content/docs/rosa/prereq-list/index.md

```diff
@@ -0,0 +1,103 @@
+  - Verify your credentials by running `rosa whoami`.
+  - Ensure you have sufficient quota by running `rosa verify quota`.
+      - Please refer [here](https://docs.openshift.com/rosa/rosa_planning/rosa-sts-aws-prereqs.html#rosa-aws-policy-provisioned_rosa-sts-aws-prereqs) for more details on AWS services provisioned for ROSA cluster. 
+      - Please refer [here](https://docs.openshift.com/rosa/rosa_planning/rosa-sts-required-aws-service-quotas.html) for more details on AWS services quota. 
+### OpenShift CLI (`oc`):
+  - Install from [here](https://docs.openshift.com/container-platform/4.13/cli_reference/openshift_cli/getting-started-cli.html) or from the OpenShift console [here](https://console.redhat.com/openshift/downloads#tool-oc).
+  - Verify that the OpenShift CLI has been installed correctly by running `rosa verify openshift-client`.
+
+Once you have the above prerequisites installed and enabled, let's proceed to the next steps.
+
+
+## SCP Prerequisites
+It is a best practice for the ROSA cluster to be hosted in an AWS account within an AWS Organizational Unit. A [service control policy (SCP)](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_scps.html) is created and applied to the AWS Organizational Unit that manages what services the AWS sub-accounts are permitted to access. 
+
+- Ensure that your organization's SCP are not more restrictive than the roles and policies required by the cluster.
+
+- Ensure that your SCP is configured to allow the required `aws-marketplace:Subscribe` permission when you choose `Enable ROSA` from the console, and please refer [here](https://docs.aws.amazon.com/ROSA/latest/userguide/troubleshoot-rosa-enablement.html#error-aws-orgs-scp-denies-permissions) for more details.
+
+- When you create a ROSA cluster using AWS STS, an associated AWS OpenID Connect (OIDC) identity provider is created as well. 
+    - This OIDC provider configuration relies on a public key that is located in the `us-east-1` AWS region. 
+    - Customers with AWS SCPs must allow the use of the `us-east-1` AWS region, even if these clusters are deployed in a different region.
+
+
+## Networking Prerequisites
+Next, let's talk about the prerequisites needed from networking standpoint.
+
+### Firewall
+  - Configure your firewall to allow access to the domains and ports listed [here](https://docs.openshift.com/rosa/rosa_planning/rosa-sts-aws-prereqs.html#osd-aws-privatelink-firewall-prerequisites_rosa-sts-aws-prereqs)
+### Custom DNS
+  - If you want to use custom DNS, then ROSA installer must be able to use VPC DNS with default DHCP options so it can resolve hosts locally. 
+      - To do so, run `aws ec2 describe-dhcp-options` and see if the VPC is using VPC Resolver.
+      - Otherwise, the upstream DNS will need to forward the cluster scope to this VPC so the cluster can resolve internal IPs/services.
+
+## PrivateLink Prerequisites
```

Comment:
non private-link VPCs also support BYO-VPC.  we should call out those requirements separately.

Reply:
i'll add a note at the end